pkgs/applications/audio/hydrogen/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Advanced drum machine";
     homepage = "http://www.hydrogen-music.org";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```

Comment:
Please do not just replace `gpl2` with `gpl2Only`: check whether the project is `gpl2Only` or `gpl2Plus`. All GPL-licensed files in Hydrogen are GPL 2 or later, [like this](https://github.com/hydrogen-music/hydrogen/blob/1.0.2/src/core/src/audio_engine.cpp#L10), therefore it is `gpl2Plus`.

Reply:
I actually checked what the project license was (I don't blindly merge stuffs). But I didn't understand correctly the meaning of `gpl2Only`: since Github **only** report the GPL2 license, i choose `gpl2Only`...
So, I now better understand the meaning of these attributes. Sorry for the mistake.

Reply:
Addressed in https://github.com/NixOS/nixpkgs/pull/120154